openedx/features/course_duration_limits/config.py

```diff
@@ -17,3 +24,36 @@
     flag_name=u'studio_ui',
     flag_undefined_default=False
 )
+
+EXPERIMENT_ID = 11
+EXPERIMENT_DATA_HOLDBACK_KEY = 'holdback_{0}'
+
+
+@receiver(ENROLL_STATUS_CHANGE)
+def set_value_for_content_type_gating_holdback(sender, event=None, user=None, **kwargs):  # pylint: disable=unused-argument
```

Comment:
@cpennington is this the right place for this code to go?

Reply:
Yeah, looks right.